css-structure/css-structure_selectors-and-cascading/css/styles.css

```diff
@@ -49,93 +49,79 @@ nav li {
 /* -------------------------------------------------------- */
 
 /* 1. These styles should apply to the headline */
-FIX_ME {
-  outline: 2px dashed red;
+h1 {
   font-size: 2rem;
-  margin: 0 2rem 0 0;
+  margin: 0;
   color: #2673bb;
 }
 
 /* 2. These styles should apply to all links */
-FIX_ME {
-  outline: 2px dashed red;
+a {
   color: #ff00e6;
 }
 
 /* 3. These styles should apply to all external links (you can use the attribute selector) */
-FIX_ME {
-  outline: 2px dashed red;
+[target] {
```

Comment:
check out: https://developer.mozilla.org/en-US/docs/Web/CSS/Attribute_selectors
you might want to use an attribute selector here: `a[href^="http"]`
This will select all links starting with `http`

Reply:
Fixed it, thank you very much.